config/etfuturum/functions.cfg

```diff
@@ -0,0 +1,278 @@
+# Configuration file
+
+##########################################################################################################
+# changes
+#--------------------------------------------------------------------------------------------------------#
+# Changes to existing content.
+##########################################################################################################
+
+changes {
+    # Adds a packet to send the attackedAtYaw field value to the client, which allows the screen to tilt based on where damage came from, and either left or right for direction-less sources like drowning or burning, instead of tilting to the left no matter what. [default: true]
+    B:enableAttackedAtYawFix=true
+
+    # Dead Bushes drop sticks [default: true]
+    B:enableBushSticks=true
+
+    # Backports recipe buffs to doors (from 1 to 3) [default: true]
+    B:enableDoorRecipeBuffs=true
+
+    # Fences, gates and dead bushes burn [default: true]
+    B:enableExtraBurnableBlocks=true
+
+    #  [default: true]
+    B:enableFloatingTrapDoors=true
+
+    # If true, fall damage on a hay bale will be reduced [default: true]
+    B:enableHayBaleFalls=true
+
+    # Allows blocks like hay bales, leaves etc to mine faster with hoes [default: true]
+    B:enableHoeMining=true
+
+    #  [default: true]
+    B:enableShearableCobwebs=true
+
+    # Mushroom blocks can be silk-touched [default: true]
+    B:enableSilkMushroom=true
+
+    # Skulls drop from charged creeper kills [default: true]
+    B:enableSkullDrop=true
+
+    # Makes mossy, cracked and chiseled stone brick craftable [default: true]
+    B:enableStoneBrickRecipes=true
+
+    # Use updated food values [default: true]
+    B:enableUpdatedFood=true
```

Comment:
should be false, right? this would just fight with out other food mods

Reply:
Done.

Reply:
> should be false, right? this would just fight with out other food mods

This only adjusts the values of specific vanilla foods.